huaweicloud/services/acceptance/hss/resource_huaweicloud_hss_image_batch_scan_test.go

```diff
@@ -0,0 +1,48 @@
+package hss
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccImageBatchScan_basic(t *testing.T) {
+	// lintignore:AT001
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccImageBatchScan_basic,
```

Comment:
```suggestion
				Config: testAccImageBatchScan_basic,
				Config: testAccImageBatchScan_operateAll,
```
Write together, one TestStep is enough.

Reply:
These are two resources with different resource names, and cannot use the same `TestStep`